angular-ngrx-scss/src/app/state/user/user.actions.ts

```diff
@@ -12,3 +13,33 @@ export const fetchUserDataError = createAction(
   '[User API] User Data fetch unsuccessful',
   props<{ error: object }>(),
 );
+
+export const getUserGists = createAction(
```

Comment:
Can we change these to use "fetch" instead of "get"? To keep the naming consistent throughout?

- [ ] Rename these to use "fetch" instead of "get" (all new actions)